### DISCUSSION TITLE
OnePub path error - terminal restart required

### DISCUSSION THREAD

**Discussion by furkanayyildiz** [onepub-dev/onepub#387] [Q&A] [answered]
I try install onepub to my project but i got this warning

**Warning: Pub installs executables into C:\Users\furka\AppData\Local\Pub\Cache\bin, which is not on your path.
You can fix that by adding that directory to your system's "Path" environment variable.**

I added to my environment variables but i have still this warning

**Comment by bsutton** [comment, marked as answer]
Did you restart you terminal after changing the environment variable?

**Reply by furkanayyildiz** [reply]
ohh !! I worked same terminal
I opened new terminal and it solved.
thanks

**Comment by bsutton** [comment]
Can you run:

*C:\Users\furka\AppData\Local\Pub\Cache\bin\onepub doctor *

*And report the results*
